backend/src/test/java/ca/bc/gov/hlth/mohums/util/AuthorizedClientsParserTest.java

```diff
@@ -0,0 +1,43 @@
+package ca.bc.gov.hlth.mohums.util;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class AuthorizedClientsParserTest {
+
+    @Test
+    public void testParse() {

```

Comment:
I initially had a test for parsing a live JWT created from code in MoHUmsIntegrationTests but there's no way to assert that the list of **view-client-\*** roles won't change in the future, so I went with just a known JWT.